OpenRA.Platforms.Default/Sdl2PlatformWindow.cs

```diff
@@ -510,18 +517,21 @@ static void SetSDLAttributes(GLProfile profile)
 			}
 		}
 
-		static bool CanCreateGLWindow(GLProfile profile)
+		static bool CanCreateGLWindow(GLProfile profile, List<string> errorLog)
 		{
 			// Implementation inspired by TestIndividualGLVersion from Veldrid
 
 			// Need to create and destroy its own SDL contexts as a workaround for specific buggy drivers
-			SDL.SDL_Init(SDL.SDL_INIT_VIDEO);
+			if (SDL.SDL_Init(SDL.SDL_INIT_VIDEO) != 0)
+				errorLog.Add($"{profile}: SDL init failed: {SDL.SDL_GetError()}");
```

Comment:
If Init fails, should we clear error and return here? It doesn't seem sensible to try and create a window after that.

Reply:
https://github.com/OpenRA/OpenRA/pull/19977#discussion_r849630179